sql/org_mozilla_fenix/clients_last_seen/view.sql

```diff
@@ -3,7 +3,7 @@ CREATE OR REPLACE VIEW
 AS
 WITH with_days_since AS (
   SELECT
-    -- We cannot use UDFs in a view, so we paste the body of udf_bitpos(bits) literally here.
+    -- We cannot use UDFs in a view, so we paste the body of udf.bitpos(bits) literally here.
```

Comment:
we actually can, but `udf_bitpos` seems to have disappeared prior to this PR.

Reply:
cc @jklukas

Reply:
Indeed. We renamed a number of these bit and byte manipulation functions. `bitpos` is now `pos_of_trailing_set_bit`.